client/components/webpack-build-monitor/index.jsx

```diff
@@ -81,7 +81,7 @@ const store = createStore(
 );
 
 const wrapConsole = fn => ( message, ...args ) => {
-	store.dispatch( { type: CONSOLE_MESSAGE, message } );
+	new Promise().then( () => store.dispatch( { type: CONSOLE_MESSAGE, message } ) );
```

Comment:
This will crash: the `Promise` constructor requires a function argument. You can use `Promise.resolve().then...` -- that's one of the better ways to postpone the callback until the next event tick loop.